azerothcore/azerothcore-wotlk
# feat(Core/Commands): Gear

@BarbzYHOOL
OLD PR: https://github.com/IntelligentQuantum/azerothcore-wotlk/commit/fdb85ebf2d571dfbee9531449c0f8e569be17ac8

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4105

## Tests Performed:

- Tested in game
- Arch Linux

## How to Test the Changes:


1. .gear repair
2. .gear stats (PlayerSave.Stats.MinLevel = 0) Shows only average item level
3. If you want to see player stats enable this PlayerSave.Stats.MinLevel = 1


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
